packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -185,6 +187,10 @@ export const BookEventForm = ({ onCancel }: BookEventFormProps) => {
         })
       );
     },
+    onError: () => {
+
+      errorRef && errorRef.current?.scrollIntoView({ behavior: "smooth" });
```

Comment:
Could we also scroll to fields that have errors (since they could also potentially be out of view?). Might be a bit harder because it's part of the form builder though 🤔

Reply:
Maybe at a later date :P